app/views/users/form.html.erb

```diff
@@ -48,8 +48,10 @@
       <%= f.field(:password_confirmation, type: :password) %>
     </div>
   <% end %>
-  <%= f.regenerable_field(:api_key) if !@user.new_record? && can?(:regenerate_api_key, @user) %>
-  <%= f.regenerable_field(:sms_auth_code) if !@user.new_record? && can?(:regenerate_sms_auth_code, @user) %>
+  <%= f.regenerable_field(:api_key, confirm: t('setting.api_key_warning')) if
+          !@user.new_record? && can?(:regenerate_api_key, @user) %>
```

Comment:
Thanks for the cleanup. If breaking a one-line `if`, please make it a standard (3-line) `if`. 

Unfortunately we don't have a linter for ERB ...